lib/board.rb

```diff
@@ -27,12 +27,9 @@ def format_board
     formatted_board.unshift(@grid.keys).map {|row| row.join}
   end
 
-  def update_board_human(player_selection)
-    @grid[player_selection][@grid[player_selection].rindex('.')] = 'X'
-  end
-
-  def update_board_computer(player_selection)
-    @grid[player_selection][@grid[player_selection].rindex('.')] = 'O'
+  def update_board(player_selection, player_type)
+    player_type == :human ? piece = 'X' : piece = 'O'
```

Comment:
can we pass current player in this argument and use player.type or player.piece?

Reply:
done! I've updated the method to take the current_player as an argument, and then call current_player.piece inside the method to get the X or O.